client/analysis.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
```

Comment:
This file should be indented with 2 spaces

Reply:
Please use 2 space indents